crates/swc/tests/tsc-references/exportAssignmentOfExportNamespaceWithDefault.2.minified.js

```diff
@@ -2,7 +2,5 @@
 // https://github.com/microsoft/TypeScript/issues/39149
 Object.defineProperty(exports, "__esModule", {
     value: !0
-});
-const _interop_require_default = require("@swc/helpers/_/_interop_require_default"), _a = /*#__PURE__*/ _interop_require_default._(require("a"));
-(0, _a.default)();
+}), (0, /*#__PURE__*/ require("@swc/helpers/_/_interop_require_default")._(require("a")).default)();
```

Comment:
This is different from the original code, but it looks like a bugfix.

Reply:
This is the same as old code. both `_interop_require_default` and `_a` are inlined.